packages/vuexfire/src/rtdb.ts

```diff
@@ -101,11 +101,20 @@ export function firebaseAction<S, R>(
         )
         return data
       },
-      add: (target, newIndex, data) =>
-        commit(VUEXFIRE_ARRAY_ADD, { target, newIndex, data }, commitOptions),
+      add: (target, newIndex, data) => {
+        if ('__ob__' in target) {
```

Comment:
what was the reason behind this? Not using commit would prevent devtools to see it